Makefile

```diff
@@ -117,6 +117,10 @@ ko:
 ko-publish: ko
 	KO_DOCKER_REPO=${IMAGE_REPO} $(KO) publish --base-import-paths --push=${IMAGE_PUSH} -t ${TAG} ./cmd/operator
 
+ko-deploy: manifests kustomize ko
+	cd config/manager && $(KUSTOMIZE) edit set image controller="ko://github.com/shipwright-io/operator/cmd/operator"
+	$(KUSTOMIZE) build config/default | KO_DOCKER_REPO=${IMAGE_REPO} $(KO) resolve --base-import-paths --push=${IMAGE_PUSH} -t ${TAG} -f - | kubectl apply -f -
```

Comment:
`ko resolve | kubectl apply -f -` is so common it has a built-in alias in ko: [`ko apply`](https://github.com/google/ko#ko-apply)

Reply:
Relic of testing - I was using resolve to verify the built image before applying the manifest.